doc/specs/#1155 - Enhancements to list command.md

```diff
@@ -0,0 +1,70 @@
+---
+author: Demitrius Nelon @denelon
+created on: 2021-06-11
+last updated: 2021-06-11
+issue id: 1155
+---
+
+# Enhancements to List Command
+
+For [#1155](https://github.com/microsoft/winget-cli/issues/1155).
+
+## Abstract
+
+The `winget list` command was designed to display all packages installed on a users Windows 10 machine in Add / Remove Programs (ARP). The command would also show which Apps with possible upgrades were available from sources configured in the Windows Package Manager. This specification proposes enhancements to the output provided by the list command.
+
+The first enhancement is adding the values for "Available" and "Source" for every package in configured sources rather that just packages with upgrades available.
```

Comment:
Is this intended to be at all times?

Reply:
Yes, this would be the "new" default behavior.